actionmailbox/app/controllers/action_mailbox/ingresses/amazon_ses/sns_notification.rb

```diff
@@ -2,12 +2,7 @@
 
 gem "aws-sdk-sns"
 
-begin
-  require "aws-sdk-sns"
-rescue LoadError => e
-  raise LoadError,
-        "Could not load the 'aws-sdk-sns' gem. Ensure that you've added the gem to your Gemfile.", e.backtrace
-end
+require "aws-sdk-sns"
```

Comment:
I removed this since it seemed like it wasn't being done elsewhere with these type of requirements